src/main/java/baritone/utils/ExampleBaritoneControl.java

```diff
@@ -77,7 +77,7 @@ public void onSendChatMessage(ChatEvent event) {
         }
     }
 
-    public boolean runCommand(String msg) {
+    private boolean runCommand(String msg) {
```

Comment:
Impact uses this for .b

Reply:
We should find an elegant way to expose this in the API